src/core/style/expressions/viewportAggregation.js

```diff
@@ -40,7 +40,11 @@ function generateAggregattion(metadataPropertyName, global) {
             return this.property._getMinimumNeededSchema();
         }
         _getDrawMetadataRequirements() {
-            return { columns: [this.property.name] };
+            if (!global) {
```

Comment:
is this related to te issue?

Reply:
`torque()` triggers this function, and since it didn't check for `global` it made the animations slower. It is related, although the change would make sense without torque too.